pom.xml

```diff
@@ -77,7 +77,7 @@
         <hive.version>1.2.2</hive.version>
         <joda.version>2.9.6</joda.version>
         <licenses.version>5.4.10-SNAPSHOT</licenses.version>
-        <parquet.version>1.11.1</parquet.version>
+        <parquet.version>1.11.2</parquet.version>
```

Comment:
why not to the latest version 1.12.3?

Reply:
I basically used the earliest version which doesn't have the mentioned CVE. 

I can bump up to `1.12.3`, given looking at [changelog](https://github.com/apache/parquet-mr/blob/master/CHANGES.md) for 1.12.0, it seems there is nothing mentioned as backward in-compatible